src/Parser/Warrior/Fury/Modules/Spells/MissedRampage.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+
+import SPELLS from 'common/SPELLS';
+import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
+
+import SpellLink from 'common/SpellLink';
+
+const GENERATORS = [
+  SPELLS.RAGING_BLOW,
+  SPELLS.BLOODTHIRST,
+  SPELLS.EXECUTE,
+];
+
+class MissedRampage extends Analyzer {
+  missedRampages = 0;
+
+  isGenerator(spellId) {
+    return !!GENERATORS.find(generator => generator.id === spellId);
```

Comment:
Alternatively, you could make `GENERATORS` an array of IDs themselves, and use `GENERATORS.includes(spellId)` instead.